### DISCUSSION TITLE
Perf: Populate on mongodb

### DISCUSSION THREAD

**Discussion by rubiin** [mikro-orm/mikro-orm#5716] [General]
when we populate oneToMany relation on the inverse side, it seems to be hitting one query each for each item.
We can already use aggregation and map the result back to entity instance but that would mean diving into aggregationand losing all typesafety and syntatic sugar for populate.
Can we use this by default

**Comment by B4nan** [comment]
Please provide a reproduction, I don't think we use one query per result, if so, we don't need aggregations to get around this.

**Reply by rubiin** [reply]
False alarm. The logs I was seeing was actually the logs for inverside.add() method.
Can we bulk update for this operation. Doing the add or set method seems to be hitting a single query for each item.

**Reply by B4nan** [reply]
As I said, I need a reproduction, I still don't know what you mean.

**Reply by rubiin** [reply]
sure , will provide a minimal repro
